src/main/webapp/WEB-INF/static/angular-templates/geojson/geojson-modal.js

```diff
@@ -3,12 +3,23 @@
 
 	const module = angular.module('manageTrialApp');
 
-	module.controller('GeoJSONModalCtrl', ['$scope', 'studyContext', '$uibModalInstance',
-		function ($scope, studyContext, $uibModalInstance) {
+	module.controller('GeoJSONModalCtrl', ['$scope', 'studyContext', '$uibModalInstance', '$http',
+		function ($scope, studyContext, $uibModalInstance, $http) {
 			$scope.geoJSONEditorUrl = '/ibpworkbench/controller/brapi-fieldmap?'
 				+ 'instanceId=' + $scope.$resolve.instanceId
 				+ '&cropName=' + studyContext.cropName;
 
+			$http({
+				method: 'GET',
+				url: '/ibpworkbench/controller/help/getUrl/' + $scope.$resolve.helpModule,
+				responseType: 'text',
+				transformResponse: undefined
+			}).then(function(response){
+				$scope.helpToolUrl = response.data;
+			}, function (error){
+			});
```

Comment:
Hi @gelli-leafnode, Please create a new Angular "Help Service" with a method to get the help URL. And use that here in the controller.

Reply:
Hi @abatac , noted.